### PR TITLE
MAKING THE HRISTOV GREAT AGAIN.. by giving it slightly more ammo

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

the sniper bundle now contains 30 spare rounds again
Reasons:
1. you cannot buy more ammo, you are stuck with 15 total ammo you can crit 7 people assuming you hit every single shot, with the low fire rate this is hard enough, but then add people who press W A S and D and its even harder. with a python you can also crit someone in two (or is it three now?) shots through armor, faster and cheaper
2. it can't be paired with much, you can go sniper+hardsuit, but once your 15 shots are up you're defenceless, you can go with a webvest and have 5 more TC for either a medkit (where you would have the same ammo problem) or a pistol, but you would have no space protection, you could go with the EVA suit and have 6TC left but you would have no armor
3. nukies also only get 15 shots, which is is absolutely not enough to fight the entire station. syndies also need to fight the entire station but dont have 40TC or friends like how nukies do, they just need more to accomplish less

basically, its price outweighs the benefits, the price is fine, infact that was the initial price intended when I first PR'd it (10 would also be acceptable if someone else PR's that) but the amount of ammo is simply too low and its basically a meme weapon

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- add: MADE THE HRISTOV GREAT AGAIN! The sniper bundle now contains 20 spare bullets

### CODE REVIEW

**Comment by Jackal298** [comment]
maybe then you will make two packs of cartridges and when will 25 come out?

**Comment by JoeHammad1844** [comment]
that would be a little inconvenient

**Comment by EmoGarbage404** [comment]
was this not nerfed like yesterday?

Even still, 30 shots is quite high for a sniper rifle with high-damage armor piercing bullets. Even taking into consideration that it has a lower firerate, it's quite high potential damage when compared to a lot of other options at that price range.

**Comment by Emisse** [comment]
i think you should change this to an ammo purchase instead of just giving it more ammo off the rip

**Comment by JoeHammad1844** [comment]
im trying to balance this around having the bloodred, gun and at least 30 total ammo. maybe reduce the price to 10TC, and make a big ammo 2TC?

**Comment by JoeHammad1844** [comment]
but then you would have MORE ammo than before, 45 total

**Comment by JoeHammad1844** [comment]
it has the python problem, back when it was 13TC

**Comment by Emisse** [comment]
nah dawg price is fine for it it does not need ammo base buff

**Comment by JoeHammad1844** [comment]
ill take the two ammo boxes idea

**Comment by JoeHammad1844** [comment]
there, 25 total ammo

**Comment by LankLTE** [comment]
gonna be real mate it doesn't need a buff, it's fine as-is

**Comment by Emisse** [comment]
dont buff it, just make an uplink purchase for extra mags

**Comment by JoeHammad1844** [comment]
alright then, 2TC for a small box?

**Comment by JoeHammad1844** [comment]
after we agree on price im gonna close this and make a new one

**Comment by JoeHammad1844** [comment]
ill just close it
